packages/mui-icons-material/package.json

```diff
@@ -28,8 +28,9 @@
     "url": "https://opencollective.com/mui-org"
   },
   "scripts": {
-    "build": "shx cp -r lib/ build/ && pnpm build:typings && pnpm build:copy-files",
-    "build:lib": "pnpm build:node && pnpm build:stable",
+    "build": "shx cp -r lib/ build/ && pnpm build:copy-files && pnpm build:esm-pkg",
+    "build:esm-pkg": "node ./scripts/create-esm-package-json.mjs",
+    "build:lib": "pnpm build:node && pnpm build:stable && rm lib/esm/package.json && pnpm build:typings",
```

Comment:
What creates the `lib/esm/package.json` file that we need to remove? Can we stop creating it?

Reply:
The build script creates it. The icons package is a bit special as it bakes built files into the repo. This package.json conflicts a bit with the aliasing setup for the docs. I didn't necessarily want to add a special case to the build for a single package, but we could.

Reply:
> I didn't necessarily want to add a special case to the build for a single package, but we could.

That's ok, I was just checking 👍🏼 let's go with whatever option you think it's better.

Reply:
I'm adding a `--skipEsmPkg`.